fecgov/openFEC
# Feature/endpoints testing

## Summary (required)
re-prepare this PR as requested: endpoints_walk.py added. locustfile.py updated. requirements-dev.txt file updated with 'click' package.
- Resolves # [_Issue number_]
enhance local testing - no specific issues targeted.
_Include a summary of proposed changes._

## How to test the changes locally
update your local python libraries to make sure 'click' installed.
pip install -r requirements-dev.txt

then you can run:
python endpoints_walk.py
run:
python endpoints_walk.py --help
for help options

## Impacted areas of the application
List general components of the application that this PR will affect:
local api testing
-  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
